### PR TITLE
Document new source check_repogpg flag and gpgkeys field.

### PR DESCRIPTION
**Pull request by bcl**
Documentation for https://github.com/osbuild/osbuild-composer/pull/3230

### CODE REVIEW

File: osbuild-composer/src/user-guide/managing-repositories.md [hunk 1]
```diff
@@ -45,6 +45,105 @@ This source will be used for any requests that specify fedora-32, eg. listing
...
+`createrepo_c` on your repository. For example:
+
+```bash
+cd repo/
+createrepo_c .
+cd repodata/
+gpg -u YOUR-GPG-KEY-EMAIL --yes --detach-sign --armor repomd.xml
+```
+
+In order to check this signature you need, and you need to tell osbuld-composer
```
Comment (ochosi) [inline osbuild-composer/src/user-guide/managing-repositories.md:62]: There's a typo and it seems one superfluous 'you need' (or something is missing in the sentence)
```suggestion
In order to check this signature you need, and you need to tell osbuild-composer
```

**Review by ochosi — CHANGES_REQUESTED** [review]
Overall looks good to me! Just that one sentence is a bit off.

File: osbuild-composer/src/user-guide/managing-repositories.md (hunk 1 above)
Comment (bcl) [reply osbuild-composer/src/user-guide/managing-repositories.md]: Good catch!

File: osbuild-composer/src/user-guide/managing-repositories.md [hunk 2]
```diff
@@ -45,6 +45,105 @@ This source will be used for any requests that specify fedora-32, eg. listing
...
+`createrepo_c` on your repository. For example:
+
+```bash
+cd repo/
+createrepo_c .
+cd repodata/
+gpg -u YOUR-GPG-KEY-EMAIL --yes --detach-sign --armor repomd.xml
+```
+
+In order to check this signature you need to tell osbuld-composer what gpg key
```
[the remark this answers is not in the document]
Comment (ochosi) [reply osbuild-composer/src/user-guide/managing-repositories.md:62]: While @dvdhrm is away, I guess I need to keep an eye out for these ;)

**Review by ochosi — CHANGES_REQUESTED** [review]

**Review by ochosi — APPROVED** [review]
LGTM!

**Pull request merged**